src/main/java/com/alrex/parcool/common/action/impl/Dodge.java

```diff
@@ -139,7 +139,7 @@ public void onStartInLocalClient(PlayerEntity player, Parkourability parkourabil
 			successivelyCount++;
 		}
 		successivelyCoolTick = MAX_COOL_DOWN_TICK * 3;
-		dodgeVec = dodgeVec.scale(player.getBbWidth() * ParCoolConfig.CONFIG_CLIENT.dodgeSpeedModifier.get());
+		dodgeVec = dodgeVec.scale(.6 * ParCoolConfig.CONFIG_CLIENT.dodgeSpeedModifier.get());
```

Comment:
Why do you need to use constant value, not depends on player size?

Reply:
I see. Hm but How should we deal with other mod which change player size without such carefulness (even I don't know whether such mod exists)

Reply:
Ok we should ignore such mods